misk-hibernate/src/test/kotlin/misk/hibernate/MoviesTestModule.kt

```diff
@@ -17,7 +17,7 @@ class MoviesTestModule : KAbstractModule() {
     install(Modules.override(MiskServiceModule()).with(FakeClockModule()))
     install(EnvironmentModule(Environment.TESTING))
 
-    val config = MiskConfig.load<MoviesConfig>("moviestestmodule", Environment.TESTING)
+    val config = MiskConfig.load<MoviesConfig>("moviestestmodule_vitess", Environment.TESTING)
```

Comment:
this seems off

Either Vitess should be the default and the file should be called moviestestmodule, or Vitess shouldn’t be the default and we should have a separate thing called `MoviesVitessTestModule`

Reply:
Yeah you're right. Shall I just make Vitess the default?

We could run some form of Travis matrix build too and run tests against both Vitess and MySQL if we want to.

Reply:
Note to self: we can use the Travis `env` configuration option to permute on a "database" option for our build matrix. https://docs.travis-ci.com/user/customizing-the-build/#Build-Matrix

Long term I think we should only run tests on Vitess though but maybe intermediately we should support both Vitess and MySQL?

Reply:
Made Vitess default for now. Later on I can add a MySQL build matrix dimension.

Reply:
Perfect.